torch/distributed/distributed_c10d.py

```diff
@@ -316,7 +315,7 @@ def _get_default_group():
     if not is_initialized():
         raise RuntimeError("Default process group has not been initialized, "
                            "please make sure to call init_process_group.")
-    return _default_pg
+    return group.WORLD
```

Comment:
Nit (was also true before this PR): It sort of seems like this function and `_check_default_pg` sort of do the same thing, return the default pg or throw if it is None. Should we dedupe them?